lib/guardian/db.ex

```diff
@@ -225,13 +225,6 @@ defmodule Guardian.DB do
     {:ok, amount_deleted}
   end
 
-  @doc false
-  def repo do
```

Comment:
Re-reading your PR it sounds like renaming this function and the config option to something other than `repo` would be the solution.

Adapter would suffice :+1: